internal/observability/events/event.go

```diff
@@ -0,0 +1,51 @@
+package event
```

Comment:
The package name and directory name do not match.

Reply:
fixed. ty!